### ISSUE TITLE
Version for Ctrader

### ISSUE THREAD

**Issue by seong1995** [EarnForex/PositionSizer#7]
Hi,

Hope can have version for Ctrader, because this tool is good.

**Comment by EarnForex** [comment]
I am not sure that it is codeable in cTrader right now. Besides, cTrader can calculate position size based on risk in the order form, no?

**Comment by seong1995** [comment]
![image](https://user-images.githubusercontent.com/69965367/90769949-a338f180-e323-11ea-9627-9df8fcae5922.png)

but this tool is better than ctrader build in order form. can calculate faster, more easy and also can have a look the risk all position.

 is ok if not possible, just a suggest.

**Comment by MuttleyBerks** [comment]
Interestingly, some months ago I started writing a simplified version for cTrader having been inspired by the MT5 version. cTrader does include a good order form, but only the SL/TP can be set using a % risk, the lots can not. So there is a need for it and I recently saw a survey from cTrader asking about priority of new features and one of them was placing an order based on % risk. So clearly a lot of people asked. It's one reason why I have to use MT5 vs cTrader. cTrader is a nicer more modern UI but lack of % risk order form is a pain.
